frontend/src/pages/auth/components/not-found/not-found.tsx

```diff
@@ -0,0 +1,26 @@
+import { useRouteError } from 'react-router-dom';
+
+import styless from './not-found.module.scss';
+
+interface RouteError {
+  statusText?: string;
+  message?: string;
+}
+
+const ErrorPage = (): JSX.Element => {
+  const error = useRouteError() as RouteError;
```

Comment:
```suggestion
```
don't need it for it. create just a not-found page to handle unknown routes. that it